architecture/integration-service.md

```diff
@@ -144,16 +139,12 @@ The `test.appstudio.openshift.io/kind` annotation is an optional annotation that
 
 #### Workspace
 
-The Integration service will provide Tekton workspaces to the individual PipelineRuns to supply additional resources required for testing, such as secure credentials for accessing test Environments.
+The Integration service will provide Tekton workspaces to the individual PipelineRuns to supply additional resources required for testing.
```

Comment:
I think we can remove the entire `Workspace` section since the integration service stopped providing it entirely.